### ISSUE TITLE
onTap to dismiss without Slide to dismiss

### ISSUE THREAD

**Issue by zainalazfar** [letsar/flutter_slidable#63]
I want to have an onTap to dismiss without SlideToDismiss because I dont want user to slide for dismiss. As for now, I only can enable the state.dismiss() if the state.dismissible is true. But state.dismissible is only true if slideToDismiss is not null. Please help to make state.dismiss() without slide to dismiss. Thanks

**Comment by Xgamefactory** [comment]
that would be nice to have a function on controller for example to programmatically fire dismiss with animation (trig a slide to dismiss action).

**Comment by anthonysette** [comment]
I would really like this too! I think tomorrow I am going to give it a shot. Not sure if you all would still like haha but I'll make a pull request and hope for the best!

**Comment by timekone** [comment]
If anyone is still interested I've implemented this in my fork. Branch [dismiss-on-tap](https://github.com/timekone/flutter_slidable/tree/feature/dismiss-on-tap).
To make `Slidable` non-dismissible by sliding, set the `dragDismissible` parameter of the `SlidableDismissal` constructor to `false`.

```dart
Slidable(
  ...
  dismissal: SlidableDismissal(
    dragDismissible: false,
    child: SlidableDrawerDismissal(),
    onDismissed: ...
  ),
);
```

**Comment by rgalich** [comment]
Hello,

Is it possible to create a version with this evolution?

**Comment by timekone** [comment]
> Hello,
> 
> Is it possible to create a version with this evolution?

Hi! Excuse me, I'm afraid I don't quite understand you.
My pull request was accepted and this feature is available since version 0.5.4.

@zainalazfar Can this issue be closed?

**Comment by rgalich** [comment]
It's good I managed to use the modification of zainalazfar.
You can close.
